_data/projects/OskiBot.yml

```
@@ -0,0 +1,10 @@
+name: OskiBot
+desc: An all-purpose chatbot for UC Berkeley students built on Cisco's Webex Teams messaging application
+site: https://github.com/mbzhu1/oskibot
+tags:
+  - up-for-grabs
```

These links should help with describing the project, to help with searching. I'm going to add some suggestions based on skimming the project, but feel free to choose your own

```suggestion
  - chatbot
```